mdal/frmts/mdal_hdf5.hpp

```diff
@@ -66,6 +66,7 @@ class HdfFile
       Create
     };
     typedef HdfH<H5I_FILE> Handle;
+    typedef std::shared_ptr<Handle> SharedHandle;
```

Comment:
do we need new `typedef` personally for me it is easier to read the code when shared_ptrs are expanded everywhere.